promql/engine.go

```diff
@@ -444,6 +444,7 @@ func (ng *Engine) exec(ctx context.Context, q *query) (v parser.Value, w storage
 					f = append(f, k, v)
 				}
 			}
+			f = append(f, "span", opentracing.SpanFromContext(ctx))
```

Comment:
I think we only want the span id

Reply:
and only if tracing is enabled (e.g. there is a span)

Reply:
Updated. I found some discussion around the opentracing api itself not facilitate getting this info, but jaeger does.

Reply:
Yes, we needed to wait for jaeger to get the information.